css/style.css

```diff
@@ -366,6 +366,13 @@ video {
   align-items: center;
   justify-items: center;
 }
+
+.button-gallery__text{
```

Comment:
BEM!!!!!!!!!

Reply:
why do you need this if buttons have standard text properties?

Reply:
ok, I was made that class because when I write text in that tag <a> the text go up and also go to the left, I made the porperties to center the text and erase the sub-line :D